packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -194,9 +196,32 @@ export interface IIntervalHelpers<TInterval extends ISerializableInterval> {
 		intervalType: IntervalType,
 		op?: ISequencedDocumentMessage,
 		fromSnapshot?: boolean,
+		stickiness?: IntervalStickiness,
 	): TInterval;
 }
 
+export enum IntervalStickiness {
+	/**
+	 * Interval expands to include segments inserted adjacent to the left
+	 */
+	Left = 0,
```

Comment:
Does the browser DOM use Right/Left terminology?

(I know some APIs avoid it due to RTL languages.)